cpp/collective/rendezvous/src/socket.cpp

```diff
@@ -514,10 +515,8 @@ bool SocketListenOp::tryListen(const ::addrinfo &addr) {
     // Here we follow the recommendation of Microsoft and use the non-standard
     // SO_EXCLUSIVEADDRUSE flag instead.
     if (!socket_->enableExclusiveAddressUse()) {
-        xoscar_WARNING(
```

Comment:
I found a `xoscar_WARNING` defined in [cpp/collective/rendezvous/include/socket.h](https://github.com/xorbitsai/xoscar/pull/51/files#diff-819cfc9e2984ed2c8a4b6266db58acfd9b7310d0dfa492d697f4a230a7d0a813). Why changing this `xoscar_WARNING` to std::cout?